metricbeat/module/kafka/_meta/config.yml

```diff
@@ -1,7 +1,10 @@
 - module: kafka
   #metricsets:
+  #  - broker
   #  - partition
+  #  - consumer
   #  - consumergroup
+  #  - producer
   period: 10s
   hosts: ["localhost:9092"]
```

Comment:
We may need a similar separation as the one suggested above for the reference config. Or we could have just one `config.yml` with the suggestions mentioned before and remove the `config.reference.yml`.